src/module/linux/netlink.h

```diff
@@ -0,0 +1,9 @@
+#pragma once
+
+#define NETLINK_KRF 28
+#define NETLINK_MYGROUP 28
+#define KRF_NETLINK_BUF_SIZE 48
```

Comment:
Is `48` here a known limit in netlink, or something that you've chosen? If in either case we should document it (and maybe come up with a message-packing format, since some messages might get close to that).

Reply:
It's random

Reply:
Gotcha. I'd just add a small comment saying that it's arbitrary, then.

Reply:
Boosted it to 1024, but it doesn't matter much

Reply:
Will do. Also boosted it to 1024 because why not, no need for unnecessary truncation, right?